spec/services/provider_interface/application_data_export_spec.rb

```diff
@@ -99,9 +99,18 @@ def expect_row_to_match_application_choice(row, application_choice)
     end
   end
 
-  describe 'replace_smart_quotes' do
+  describe '.replace_smart_quotes' do
     it 'replaces smart quotes in text' do
       expect(described_class.replace_smart_quotes(%(“double-quote” ‘single-quote’))).to eq(%("double-quote" 'single-quote'))
     end
   end
+
+  describe '.rejection_reasons' do
+    let(:application_choice) { create(:application_choice, :with_structured_rejection_reasons) }
+
+    it 'returns a list of rejection reasons' do
+      expect(described_class.rejection_reasons(application_choice))
```

Comment:
You could split the expected output on double newline and either write multiple assertions or assert the resulting split text array. It might be more readable.

Reply:
Yeah I'll mess around with the formatting to try and chunk it up a bit